compose-table/src/main/java/org/dhis2/composetable/ui/DataSetTableUi.kt

```diff
@@ -456,6 +467,12 @@ fun ItemValues(
                     },
                     onClick = onClick
                 )
+                if (isSelected) {
+                    val circleCoordinates = Rect(0f, 0f, 240f, 240f)
```

Comment:
What is does this 240 mean?

Reply:
It determines the rectangle that should be displayed on the view. It work like a margin for the cell when calling the bringIntoView method